language/move-binary-format/Cargo.toml

```diff
@@ -17,10 +17,10 @@ proptest = { version = "1.0.0", optional = true }
 proptest-derive = { version = "0.3.0", optional = true }
 ref-cast = "1.0.6"
 variant_count = "1.1.0"
-diem-workspace-hack = { path = "../../common/workspace-hack" }
 move-core-types = { path = "../move-core/types" }
```

Comment:
can you specify a version here as well as a path (like is done in the diem-types crate) as well as in the other toml file for the path dependnecies?

Reply:
Updated!